### PR TITLE
SWC-6183

### PR DESCRIPTION
**Pull request by yjyoo773**

### CODE REVIEW

File: src/lib/containers/EntityBadgeIcons.tsx [hunk 1]
```diff
@@ -149,7 +149,6 @@ export const EntityBadgeIcons = (props: EntityBadgeIconsProps) => {
   useEffect(() => {
     if (isInExperimentalMode && schemaValidationResults) {
       // Because we may not know to render an icon until after fetching data (i.e. after a delay), we have to rebuild the tooltip to ensure it appears
```
Comment (jay-hodgson) [inline src/lib/containers/EntityBadgeIcons.tsx:151]: This comment is now either inaccurate (and should be removed), or we need to find another way to rebuild the tooltip after the data has been fetched

File: src/lib/containers/EntityBadgeIcons.tsx [hunk 2]
```diff
@@ -243,73 +291,97 @@ export const EntityBadgeIcons = (props: EntityBadgeIconsProps) => {
...
+                placement="right"
+              >
+                <LocalOfferTwoTone
+                  aria-hidden={false}
+                  role={canOpenModal ? 'button' : 'img'}
+                  className={`EntityBadge__Badge ${schemaConformance}`}
+                  style={canOpenModal ? { cursor: 'pointer' } : undefined}
+                  onClick={canOpenModal ? () => setShowModal(true) : undefined}
+                  data-for={ownTooltipId}
+                  data-tip={sanitizeHtml(annotationsHtml)}
```
Comment (jay-hodgson) [inline src/lib/containers/EntityBadgeIcons.tsx:350]: Is this still used?  (data-tip, data-for, ...)

Comment (jay-hodgson) [reply src/lib/containers/EntityBadgeIcons.tsx:350]: Yes, it is.

Comment (jay-hodgson) [reply src/lib/containers/EntityBadgeIcons.tsx:350]: It looks like the actual tooltip comes from Tooltip.title.  Do all of these old values need to be set?

Comment (jay-hodgson) [reply src/lib/containers/EntityBadgeIcons.tsx:350]: Is annotationsHtml referenced after using annotationsHtmlX?

File: src/lib/containers/AddToDownloadListV2.tsx [hunk 3]
```diff
@@ -44,22 +44,21 @@ const AddToDownloadListV2: React.FunctionComponent<
...
-          place="right"
-          type="dark"
-          effect="solid"
-          id={`${entityId}_${entityVersionNumber}_download-list-v2-button`}
-        />
-        <IconSvg options={{ icon: 'addToCart' }} />
-      </a>
+        <a
+          data-testid="AddToDownloadListV2"
+          data-for={`${entityId}_${entityVersionNumber}_download-list-v2-button`}
```
Comment (jay-hodgson) [inline src/lib/containers/AddToDownloadListV2.tsx:55]: Is data-for still necessary, or can it be removed across this PR?

Comment (jay-hodgson) [reply src/lib/containers/AddToDownloadListV2.tsx:55]: And if not, can we remove most of the tooltip id values too?

File: src/lib/containers/EntityBadgeIcons.tsx (hunk 2 above)
Comment (yjyoo773) [reply src/lib/containers/EntityBadgeIcons.tsx:350]: > Is annotationsHtml referenced after using annotationsHtmlX?

No those aren't used. I must have forgotten to remove it.

Comment (yjyoo773) [reply src/lib/containers/EntityBadgeIcons.tsx:350]: As for data-tip, data-for those are used for tests. Ones that don't I have removed.

File: src/lib/containers/entity_finder/details/view/DetailsView.tsx [hunk 4]
```diff
@@ -552,13 +550,6 @@ export const DetailsView: React.FunctionComponent<DetailsViewProps> = ({
           </BaseTable>
         )}
       </AutoResizer>
-      <ReactTooltip
```
Comment (jay-hodgson) [inline src/lib/containers/entity_finder/details/view/DetailsView.tsx:555]: Oh I see, the entity icons used to share a single ReactTooltip

File: src/lib/containers/personal_access_token/AccessTokenCard.tsx [hunk 5]
```diff
@@ -26,12 +26,6 @@ export const AccessTokenCard: React.FunctionComponent<AccessTokenCardProps> = ({
 
   const isExpired = accessToken.state === 'EXPIRED'
 
-  useEffect(() => {
-    // For reasons unknown, the tooltips in this component would not load in Synapse.org without this
-    // see https://github.com/wwayne/react-tooltip/issues/344
-    ReactTooltip.rebuild()
```
Comment (jay-hodgson) [inline src/lib/containers/personal_access_token/AccessTokenCard.tsx:32]: Anyone else worried about the integration?
Could you create the umd build locally with these changes, and drop them into a running SWC client to get a preview of the integration (see if the tooltips are still working)?  Let me know if you need help with this, and I can show how to do this from my machine

**Review by jay-hodgson — APPROVED** [review]

**Pull request merged**